test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -5799,13 +5876,17 @@ BOOST_AUTO_TEST_CASE(warn_unused_function_parameter)
 		}
 	)";
 	CHECK_WARNING(text, "Unused function parameter. Remove or comment out the variable name to silence this warning.");
-	text = R"(
+}
+
+BOOST_AUTO_TEST_CASE(unused_unnamed_function_parameter)
+{
+	char const* text = R"(
 		contract C {
-			function f(uint a) pure public {
+			function f(uint) pure public {
```

Comment:
This test case was duplicated. There is the other one above this called "warn_unused_function_parameter".